news/2018-03-13-project-loved-week-of-march-11th.md

```diff
@@ -0,0 +1,244 @@
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/catch/eastward.jpg)](https://osu.ppy.sh/community/forums/topics/715057)
+
+*written by [JBHyperion](https://osu.ppy.sh/users/4879508)*
+
+Turning the clock back slightly to the 2016 osu!catch World Cup, [Deif](https://osu.ppy.sh/users/318565)'s rendition of [ginkiha's "eastward"](https://osu.ppy.sh/beatmapsets/365365) is an excellent combination of aesthetics and technicality. Crafting an outstanding balance between flowing streams, snappy jumps and clever wub rhythm usage, this map will serve as an exciting challenge to more experienced players seeking a rounded test of their skills.
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/catch/touhou-last-boss-rush.jpg)](https://osu.ppy.sh/community/forums/topics/715058)
+
+*written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+An [incredible marathon map](https://osu.ppy.sh/beatmapsets/286386) by [Kyuare](https://osu.ppy.sh/users/2705312) with over 20 minutes of content. By the length alone it is already obvious that a lot of work has been put into it. The difficulty is varying throughout the marathon, but overall it is very challenging to get an FC or close to an FC on this map due to the length and the consistency required. It is sad to see that such a big project has been in the graveyard for roughly 2 years now and it would certainly deserve to be put into the loved category.
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/catch/no-title.jpg)](https://osu.ppy.sh/community/forums/topics/715059)
+
+*written by [Sartan](https://osu.ppy.sh/users/4100941)*
+
+Man, have we seen [this map](https://osu.ppy.sh/beatmapsets/230741) A LOT. And yet again here it is in the loved selection. This one has been used in a multitude of tournaments and has also been a very famous pick in multiplayer lobbies for as long as it has existed. The structure of this map is very much like you would imagine from [Spectator](https://osu.ppy.sh/users/702598): very clean, great emphasis on the music (al changes throughout the song), no uncomfortable difficulty spikes and a calming, enjoyable song, it's replayability having been proven by the map's usage throughout various community activities. Even though we can't change the fact the song is called No title, perhaps you can help it become Yes entitled to being loved after voting!
```

Comment:
I'm kinda stuck at `music (al changes`, I'm not sure if it is a typo of I just can't find the correct definition of `al`

Reply:
I think it was supposed to be like "music-al changes" (being broken by the parenthesis). I think I'm just going to remove the parenthesis because that's pretty awkward to read